WordScramble.js

```diff
@@ -18,35 +19,35 @@ function scramble(){
     scrambledLetters = letters.join('');
     document.getElementById("scrambledword").textContent = scrambledLetters;
 }
+
+let checkButtons = false;
```

Comment:
this could be named better. `isRetryButtonRendered` - this tells me exactly what is true or false.